server/fly.io/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use axum::{
+    extract::Query,
+    routing::{get, post},
+    Router,
+};
+use std::collections::BTreeMap;
+use std::net::SocketAddr;
+
+#[tokio::main]
+async fn main() {
+    let app = Router::new()
+        .route("/", get(root))
+        .route("/record-install", get(record_install))
+        .route("/record-install", post(record_install));
+
+    // ipv6 + ipv6 any addr
+    let addr = SocketAddr::from(([0, 0, 0, 0, 0, 0, 0, 0], 8080));
+    axum::Server::bind(&addr)
+        .serve(app.into_make_service())
+        .await
+        .unwrap();
+}
```

Comment:
It's better to write it as:

```suggestion
fn main() {
    let rt = tokio::runtime::Runtime::new().unwrap();
    let task = rt.spawn(async move {
        let app = Router::new()
            .route("/", get(root))
            .route("/record-install", get(record_install))
            .route("/record-install", post(record_install));

        // ipv6 + ipv6 any addr
        let addr = SocketAddr::from(([0, 0, 0, 0, 0, 0, 0, 0], 8080));
        axum::Server::bind(&addr)
            .serve(app.into_make_service())
            .await
            .unwrap();
    });
    rt.block_on(task).unwrap();
}
```

so that the main server task can be run on any thread instead of just on `main`, which could be a performance bottleneck.